frontend/src/metabase/query_builder/components/DimensionList.jsx

```diff
@@ -111,7 +112,7 @@ export default class DimensionList extends Component {
             triggerElement={this.renderSubDimensionTrigger(
               item.dimension,
               multiSelect,
-              preventNumberSubDimensions,
```

Comment:
This was unused, and it is from [#18800](https://github.com/metabase/metabase/pull/18800/files#diff-a747d1635f99a8170f6f7e9bfd95d5a2375e80241fc1c518131ee152805d1406R147), but you could see that even [in that version](https://github.com/metabase/metabase/blob/cb707176642c01e07bac4dd7430fb38525614ee0/frontend/src/metabase/query_builder/components/DimensionList.jsx), `renderSubDimensionTrigger` still didn't accept the third argument.